pg_mon.c

```diff
@@ -439,13 +439,13 @@ pgmon_save_firsttuple(QueryDesc *queryDesc)
 static void
 pgmon_plan_store(QueryDesc *queryDesc)
 {
+        Assert(queryDesc != NULL);
```

Comment:
It is illegal in C to have a variable declaration not at the beginning of code block.